app/js/components/Buyer/SorterFilter.jsx

```diff
@@ -1,18 +1,91 @@
-import React from 'react';
+import React, {Component, Fragment} from 'react';
 import PropTypes from 'prop-types';
 import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
 import {faAngleDown, faSortAmountDown} from "@fortawesome/free-solid-svg-icons";
+import classnames from 'classnames';
 
 import './SorterFilter.scss';
+import CheckButton from '../ui/CheckButton';
+import {ButtonGroup, FormGroup} from "reactstrap";
+import {Typeahead} from "react-bootstrap-typeahead";
 
-const SorterFilter = () => (
-  <div className="sorter-select font-weight-bold px-3 py-2 bg-secondary rounded v-align-center my-3">
-    <span className="sort-icon bg-dark text-white rounded-circle d-inline-block text-center p-2 mr-2">
-      <FontAwesomeIcon icon={faSortAmountDown}/>
-    </span> Sort and filter <span className="float-right pt-1"><FontAwesomeIcon size="2x" icon={faAngleDown}/></span>
-  </div>
+const FilterMenu = ({open, close}) => (
+  <Fragment>
```

Comment:
There is probably a good reason but why didn't you use modal from reactstrap?

Reply:
Really didn't think of using a modal to be honest. It never occurred to be that I could use them because of how they do not have the same shape, but I agree it could have worked, since it handles the backdrop for us. 

We will see if it bites us in the butt, but at least it gives us more control for now.